lib/mutant/cache.rb

```diff
@@ -1,14 +1,17 @@
 module Mutant
   # An AST cache
   class Cache
-    include Equalizer.new, Adamantium::Mutable
+    include Concord.new(:file), Adamantium::Mutable
 
     # Initialize object
     #
+    # @param [Class<File>]
+    #
     # @return [undefined]
     #
     # @api private
-    def initialize
+    def initialize(*)
+      super
       @cache = {}
```

Comment:
I realize this isn't part of the changed code, but does this `Hash` need to be accessed from multiple threads? If so you might be better off with a `ThreadSafe::Cache` instance here instead.